external/source/exploits/CVE-2020-0796/LPE/CVE-2020-0796.vcxproj

```diff
@@ -106,7 +106,7 @@
       </SDLCheck>

```

Comment:
I updated this file so the build paths are still correct after being moved into the LPE subdirectory. The original exploit binary is unmodified, this just means you can still open Visual Studio, hit build and things will work as intended.